libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.java

```diff
@@ -385,9 +385,7 @@ public void onPickServerClick(View v) {
 
 	@Override
 	protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-		if (requestCode == PasscodeManager.PASSCODE_REQUEST_CODE && resultCode == Activity.RESULT_OK) {
-			webviewHelper.onNewPasscode();
```

Comment:
The onNewPasscode method was taking care of creating the user account. Now the user account will already have been created.